website/static/js/pages/profile-settings-addons-page.js

```diff
@@ -135,6 +143,8 @@ $(window).on('beforeunload',function() {
 ****************/
 
 $('.addon-oauth').each(function(index, elem) {
+// Applies to Mendeley, Zotero.  There is a separate code for dataverse in DataverseUserConfig.js
```

Comment:
I think almost all of the addons are using `OAuthAddonSettingsViewModel` now  (except Dataverse and s3 I believe) so this comment isn't quite right